Resources/public/js/views/actions/ez-translateactionview.js

```diff
@@ -62,15 +62,35 @@ YUI.add('ez-translateactionview', function (Y) {
 
                 location: this.get('location').toJSON(),
                 content: this.get('content').toJSON(),
-                translations: translationsList,
-                canDeleteTranslations: translationsList.length > 1,
```

Comment:
This is potential BC break, someone could override the template and still rely on this variables.

Reply:
I agree

Reply:
Not sure to understand the BC here. Could you please provide more details @dew326 ?

Reply:
If someone had overidden the translateaction view template and relies on any variable sent to the template while rendering, then it may break the platform ui app. If you don't remove template variables then it won't break anything.

Reply:
Thanks @sunpietro